blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_9_detailling_and_specific_rules/formula_9_1n.py

```diff
@@ -57,3 +58,25 @@ def _evaluate(
         """For more detailed documentation see the class docstring."""
         raise_if_negative(f_ctm=f_ctm, f_yk=f_yk, b_t=b_t, d=d)
         return max(0.26 * (f_ctm / f_yk) * b_t * d, 0.0013 * b_t * d)
+
+    def latex(self) -> LatexFormula:
+        """Returns LatexFormula object for formula 9.1N."""
+        return LatexFormula(
+            return_symbol="A_{s,min}",
+            result=value_to_latex_text(self),
+            equation=max_curly_brackets_latex(
+                value_to_latex_text(0.26) + r"\cdot \frac{f_{ctm}}{f_{yk}} \cdot b_t \cdot d",
+                value_to_latex_text(0.0013) + r"\cdot b_t \cdot d",
+            ),
+            numeric_equation=max_curly_brackets_latex(
+                value_to_latex_text(0.26)
```

Comment:
eventhough adding the different strings might be usefull to avoid huge f-strings. I think f-strings is the default way to go. 
So I think we should find a different solution for f-strings or concatenating strings.